content/guide/switching-to-preact.md

```diff
@@ -81,6 +81,12 @@ In this case though, you might find it more compelling to switch directly to `pr
 Preact's core is quite fully featured, and many idiomatic React codebases can actually be switched straight to `preact` with little effort.
 That approach is covered in the next section.
 
+#### Notes about using it with 3rd modules
+
+It's probably that you are looking for use react to get compatibility with community React modules. If so, remember to configure the [Aliasify](https://www.npmjs.com/package/aliasify) transform with `global` (`--global-transform`). This is a [Browserify](https://github.com/browserify/browserify) option.
+
+É provável que você esteja querendo utilizar preact-compat, para conseguir usar módulos de React da comunidade. Se este é o caso, lembre-se de configurar o transform do [Aliasify](https://www.npmjs.com/package/aliasify) como `global` (`--global-transform`). Essa é uma configuração do [Browserify](https://github.com/browserify/browserify).
```

Comment:
Duplicate line?

Reply:
You're right. I've commited the correction